config/incrementals-publisher.yaml

```
@@ -12,3 +12,6 @@ ingress:
     - secretName: incrementals-tls
       hosts:
         - incrementals.jenkins.io
+
+nodeSelector:
+  agentpool: publicpool
```

```suggestion
  agentpool: x86medium
```

Use the new node pool